core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -155,6 +155,14 @@ protected boolean supportsServerSideRetry() {
     return false;
   }
 
+  protected boolean supportsServerManagedLocation() {
+    return false;
+  }
+
+  protected boolean supportsNamesWithSlashes() {
```

Comment:
For REST based catalogs, some servers don't allow escaped slashes in the path by default, like Tomcat. There are workarounds but it can open up potential security issues, so some may choose not to support this.

Reply:
Is that for escaped slashes in the typical way or for the REST catalogs escaped slashes that use `%00` (Null byte)?

I’m good with not requiring that server implementations require this (I’m almost certain it’s already not required) but would want to ensure the spec is up to date. But if the null-byte can open up security problems then I can understand people deciding tables / namespaces really don’t benefit from slashes in them.

Reply:
The null byte is used as a delimiter AFAIK, the slash is just URL encoded, i.e. `%2F`

Reply:
Sounds reasonable to me.